src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/validator/MatchPrisonersRequestValidator.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.validator
+
+import org.springframework.stereotype.Service
+import org.springframework.validation.Errors
+import org.springframework.validation.Validator
+import uk.gov.justice.digital.hmpps.welcometoprison.model.prison.prisonersearch.request.MatchPrisonersRequest
+
+@Service
+class MatchPrisonersRequestValidator : Validator {
+  override fun supports(clazz: Class<*>): Boolean {
```

Comment:
clazz?

Reply:
that's a weird but common idiom in java - you can't uses the word `class` as its a keyword so people tend to use `clazz` instead!